### PR TITLE
abstract

### PR DESCRIPTION
**Pull request by tpalinski**
nie ma pdfa, to tylko tekst

### CODE REVIEW

File: meta/abstract.tex [hunk 1]
```diff
@@ -1,4 +1,5 @@
 \chapter*{Abstract}
-\indent This paper describe.... Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus elementum arcu nec blandit aliquam. Integer eros dolor, molestie eget dictum quis, luctus sit amet sapien. Proin dignissim felis in ornare volutpat. Morbi vulputate rutrum efficitur. Ut vehicula vehicula metus, et iaculis tortor mattis vel. Nam blandit, arcu quis ultricies blandit, libero ante commodo augue, in accumsan dui leo at orci. Phasellus in augue et velit pulvinar malesuada ut et sem. Nulla vehicula nibh eu odio sollicitudin sagittis. Praesent condimentum semper neque, tincidunt luctus nisl scelerisque sed. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. 
+\indent STOS, or Student Testing and Grading System \textit{(System testowania i oceny studentów)} is a system used extensively as a part of the educational process at Faculty of Electronics, Telecommunications and Informatics at Gdańsk Tech. It acts as the backbone of the procedure of automatic grading of practical assignments submitted by students, which helps the course instructors with timely evaluation of the students' work.
```
Comment (AdamNiesiobedzki) [inline meta/abstract.tex:2]: STOS? More like STGS (or STDs because this system is not safe)

Comment (AdamNiesiobedzki) [inline meta/abstract.tex:2]: Gdańsk University of Technology?

**Review by AdamNiesiobedzki — APPROVED** [review]
I don't get a single word but 10/10

File: meta/abstract.tex (hunk 1 above)
Comment (tpalinski) [reply meta/abstract.tex:2]: On their website it's Gdańsk Tech, those are interchangeable probably

**Review by lukrzak — APPROVED** [review]
Masz lajka

**Pull request merged**